app/src/main/java/com/example/countryinfo/CountryPreviewFragment.kt

```diff
@@ -64,6 +64,7 @@ class CountryPreviewFragment : Fragment() {
                 }
             }
         })
+        viewModel.getCountries()
```

Comment:
Why this was moved there? We shouldn't change production code in order to just satisfy the tests.

Reply:
Changed.